.github/workflows/ci.yml

```
@@ -0,0 +1,38 @@
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+
+      - name: Run tests
+        run: |
+          PYTHONPATH=. pytest -v
+
+      - name: Run lint
+        run: |
+          pip install flake8
+          flake8 app tests --exit-zero
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)